aidants_connect_web/templates/public_website/section_link_faq_partial.html

```diff
@@ -0,0 +1,11 @@
+<section class="section section-color">
+  <div class="container">
+    <h3>Vous avez une question ?</h3>
+    <p>
+      Quelles sont les personnes qui peuvent bénéficier d’Aidants Connect ?
+      Quelles sont les conditions pour acquérir le statut "Aidants Professionel" ?
+      Combien coûte Aidants Connect ? etc.
```

Comment:
Légère modification de texte + correction coquilles

```suggestion
      Quelles sont les personnes qui peuvent bénéficier d’Aidants Connect ?
      Qu'est-ce qu'un "aidant professionnel" ?
      Comment fonctionne Aidants Connect ? etc.
```